numpy/core/tests/test_multiarray.py

```diff
@@ -3606,11 +3605,18 @@ def test_record_no_hash(self):
         a = np.array([(1, 2), (1, 2)], dtype='i1,i2')
         self.assertRaises(TypeError, hash, a[0])
 
+    def test_empty_structure_creation(self):
+        # make sure these do not raise errors (gh-5631)
```

Comment:
Wow, but it makes sense, why not have no fields if you have a size.... Spaces after ":" though ;).